### PR TITLE
Create SQL files PR

### PR DESCRIPTION
**Pull request by mertkantaroglu**
Hello,

In this project I created vet_clinic database and animals table. Afterward I populated the SQL files per requirements and took screenshots of queries.

Thanks.

Screen Shot 2023-04-10 at 17 37 35
Screen Shot 2023-04-10 at 18 14 34
Screen Shot 2023-04-10 at 18 19 45
Screen Shot 2023-04-10 at 18 19 55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,98 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank for Microverse to provide us learning materials, and my peers for helping me out to move forward.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENSE) licensed.
```
Comment (CollinsTatang) [inline README.md:96]: - [x] Please fix the link to your `LICENSE` file the current link is broken.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @mertkantaroglu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You have properly implement the different queries as per the project requirement :heavy_check_mark: 

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @mertkantaroglu ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mertkantaroglu** [comment]
Thank you @1GORDON 👍

**Pull request merged**